doc/user/content/overview/architecture.md

```diff
@@ -110,14 +110,12 @@ view is in terms of how long the generated dataflow persists.
   read on this data at any given time than it is to create an answer from
   scratch.
 
-The only "wrinkle" in the above explanation is when you perform reads on views:
-no dataflow gets created, and Materialize instead serves the result from an
-existing dataflow.'
+The only wrinkle in the above explanation is that when you perform reads on views, no dataflow gets created, and Materialize instead serves the result from an
+existing dataflow.
 
 ## Sources: Ingesting data
 
-For Materialize to ingest data, it must read it from a source, of which there
-are two varieties:
+For Materialize to ingest data, it must read it from a source, which comes in two varieties:
```

Comment:
This is less true now that tables exist. Sources are still a fine way to ingest data, but not the only way.